authcp/views.py

```diff
@@ -133,3 +133,16 @@ def activation(request, key):
         # Display : error message
         already_active = True
     return render(request, 'authcp/activation.html', locals())
+
+
+# save user who is registered with google or facebook
+def save_profile(backend, user, response, *args, **kwargs):
+
+    if backend.name == "github":
+        print "Github user {} is logged in".format(user)
```

Comment:
Actually there is a logging mechanism in place. If check following source:
https://github.com/palamario/palamar/blob/master/palamar/_i18n.py

You will see that logging also needs to be translated. But at first English is good enough.

I have used this feature in my code before. Check the example from following source:
https://github.com/palamario/palamar/blob/49edcc51075745243a1a585bea738bd77bccafcf/domain/views.py

You can check the logging configuration from settings.py.